doc/source/whatsnew/v0.24.0.txt

```diff
@@ -26,6 +26,33 @@ Backwards incompatible API changes
 
 .. _whatsnew_0240.api.datetimelike:
 
+Tick DateOffset Normalize Restrictions
+--------------------------------------
+
+Creating a ``Tick`` object (:class:``Day``, :class:``Hour``, :class:``Minute``,
+:class:``Second``, :class:``Milli``, :class:``Micro``, :class:``Nano``) with
+`normalize=True` is no longer supported.  This prevents unexpected behavior
+where addition could fail to be monotone or associative.
+
+.. ipython:: python
+
+    ts = pd.Timestamp('2018-06-11 18:01:14')
+    tic = pd.offsets.Hour(n=2, normalize=True)
+
+Previous Behavior:
+
+.. code-block:: ipython
+
+    In [2]: ts = pd.Timestamp('2018-06-11 18:01:14')
+
+    In [3]: tic = pd.offsets.Hour(n=2, normalize=True)
+
+    In [4]: ts + tic
+    Out [4]: Timestamp('2018-06-11 00:00:00')
+
+    In [5]: ts + tic + tic + tic == ts + (tic + tic + tic)
+    Out [5]: False
+
```

Comment:
add the Current  behavior via an ipython block (obviously w/o normalize passed)

Reply:
OK.  And just the same `ts + tic + tic + tic == ts + (tic + tic + tic)` demonstration?  With `normalize=False` that behavior is unchanged.